src/femtomes.cpp

```diff
@@ -434,3 +639,101 @@ GPSDriverFemto::calculateBlockCRC32(uint32_t length, uint8_t *buffer, uint32_t c
 	return (crc);
 }
 
+void GPSDriverFemto::activateCorrectionOutput()
+{
+    if (_correction_output_activated || _output_mode != OutputMode::RTCM)
+    {
+        return;
+    }
+
+    char buffer[100];
+
+    if (_base_settings.type == BaseSettingsType::survey_in)
+    {
+        FEMTO_DEBUG("Femto: enabling survey-in")
+
+        if(writeAckedCommandFemto("POSAVE ON \r\n","<POSAVE OK",FEMTO_RESPONSE_TIMEOUT) == 0)
```

Comment:
How long will the survey-in take and is that configurable?

Reply:
It will take a few seconds. The command POSAVE ON will reply immediately,And device will set the average position of the base station, and the position average will continue until fixed, then start RTCM3 output.

Reply:
So the averaging duration cannot be configured?

Reply:
Yes，this is currently not configurable